src/geohash_helper.c

```diff
@@ -84,32 +84,22 @@ uint8_t geohashEstimateStepsByRadius(double range_meters, double lat) {
 
 /* Return the bounding box of the search area by shape (see geohash.h GeoShape)
  * bounds[0] - bounds[2] is the minimum and maximum longitude
- * while bounds[1] - bounds[3] is the minimum and maximum latitude.
- *
- * This function does not behave correctly with very large radius values, for
- * instance for the coordinates 81.634948934258375 30.561509253718668 and a
- * radius of 7083 kilometers, it reports as bounding boxes:
- *
- * min_lon 7.680495, min_lat -33.119473, max_lon 155.589402, max_lat 94.242491
- *
- * However, for instance, a min_lon of 7.680495 is not correct, because the
- * point -1.27579540014266968 61.33421815228281559 is at less than 7000
- * kilometers away.
- *
- * Since this function is currently only used as an optimization, the
- * optimization is not used for very big radiuses, however the function
- * should be fixed. */
```

Comment:
I see this comment existed in redis 6.0 (when only GEORADIUS was supported).
are you saying that the change i asked you to re-add from the trapezoid PR (about top longitude and bottom longitude), is what solves this inaccuracy, and hence the comment can be removed?

Reply:
yes, the inaccuracy mentioned in the previous comments is that trapezoid is treated as rectangle.